packages/protocol/contracts/lib/Checkpointing.sol

```diff
@@ -4,6 +4,8 @@ pragma solidity ^0.5.17;
 /**
 * @title Checkpointing - Library to handle a historic set of numeric values
 */
+// Note: perhaps worth using the same naming / contract from
+// https://github.com/aragonone/voting-connectors/blob/master/shared/contract-utils/contracts/Checkpointing.sol?
```

Comment:
I've taken the liberty of moving over some of the optimizations I made in the Voting Connectors, but was thinking that we might as well also align on a similar interface / codebase so we don't have two very similar, but slightly different implementations of the logic.

Reply:
That makes sense,  but then ideally we should have it a separate package that we can import, right?
Besides, that one is Solidity 4.